deps/libgit2.version

```diff
@@ -3,11 +3,11 @@
 LIBGIT2_JLL_NAME := LibGit2
 
 ## source build
-LIBGIT2_BRANCH=v1.5.0
-LIBGIT2_SHA1=fbea439d4b6fc91c6b619d01b85ab3b7746e4c19
+LIBGIT2_BRANCH=v1.6.0
+LIBGIT2_SHA1=f7325c448053bfcd1a32ec319ed392f525480634
 
 ## Other deps
 # Specify the version of the Mozilla CA Certificate Store to obtain.
 # The versions of cacert.pem are identified by the date (YYYY-MM-DD) of their changes.
 # See https://curl.haxx.se/docs/caextract.html for more details.
-MOZILLA_CACERT_VERSION := 2022-10-11
+MOZILLA_CACERT_VERSION := 2023-01-10
```

Comment:
We probably want to update it on Yggdrasil (https://github.com/JuliaPackaging/Yggdrasil/blob/852acea4fd0bdc48ed72601e73ff4e087d436b6c/M/MozillaCACerts/build_tarballs.jl) and then here https://github.com/JuliaLang/julia/blob/f0eadd076f2728797cf5be88b61cda1d9f1dd515/stdlib/MozillaCACerts_jll/Project.toml#L3

Reply:
Ok, I updated the certs in Yggdrasil (https://github.com/JuliaPackaging/Yggdrasil/pull/6310), so now you can update the version number in `Project.toml` and then referesh the checksums with `make -f contrib/refresh_checksums.mk mozillacert` :slightly_smiling_face: